scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -572,10 +572,7 @@ <h6><span class="fa fa-map-pin ms-3 mt-2"></span>&nbsp;Pinned variants ({{suspec
     <ul class="list-group collapse" id="matchingCausatives">
       {% for variant in other_causatives %}
         <li class="list-group-item">
-          <a href="{{ url_for('variant.variant', institute_id=institute._id,
-                              case_name=case.display_name, variant_id=variant._id) }}">
-            {{ variant.hgnc_symbols|join(', ') or variant.hgnc_ids|join(', ') or variant.str_repid }}
-          </a>
+          {{ pretty_link_variant(variant, case)}}
```

Comment:
👍🏻

Reply:
Yeah, I think we (I) might have had the idea to make them a tad more non-informative (before the hiding div) so that the user had one more moment to think about whether they really wanted to triage a variant in that gene for that patient. Some of the CLGs would insta-recognise common managed variants.

Reply:
It should be ok with the hiding div I think. Thanks for the review!